packages/paste-website/src/components/site-wrapper/site-header/DarkModeFAQModal.tsx

```diff
@@ -64,13 +64,13 @@ export const DarkModeFAQModal: React.FC = () => {
       </Paragraph>
       <Paragraph>
         Themes are an implementation of design language. Paste, by default, uses the new design language that was
-        specifically designed to unify all Twilio products. Paste also comes with legacy Console and legacy SendGrid
-        design language support.
+        specifically designed to unify all Twilio products. Paste also comes with legacy SendGrid design language
```

Comment:
This line is a bit of a lie I guess. The SendGrid theme is the Default theme with a different type face. 

Maybe we reword this a little to something like "Paste also comes with the ability to support legacy design language support." Make it a little more general.